src/models/serw13/README.md

```diff
@@ -0,0 +1,64 @@
+# Serval WS (serw13)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Serval WS](./img/serw13.webp)
+
+The System76 Serval WS is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel 13th Generation (Raptor Lake) CPUs
+        - [Core i5-13500HX](https://ark.intel.com/content/www/us/en/ark/products/232156/intel-core-i513500hx-processor-24m-cache-up-to-4-70-ghz.html)
+        - [Core i7-13700HX](https://ark.intel.com/content/www/us/en/ark/products/232166/intel-core-i713700hx-processor-30m-cache-up-to-5-00-ghz.html)
+        - [Core i9-13900HX](https://ark.intel.com/content/www/us/en/ark/products/232171/intel-core-i913900hx-processor-36m-cache-up-to-5-40-ghz.html)
+- Chipset
+    - Intel HM770
+- BIOS
+    - GigaDevice GD25B256EYIGR
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU options:
+        - NVIDIA GeForce RTX 4050
+        - NVIDIA GeForce RTX 4060
```

Comment:
I don't see a 4050 option on this page and there isn't a 4K option listed here either.

Reply:
The serw13 will only come with 4060 and 4070 options. The addw3 will have all three.